FirebaseAnalytics.podspec

```diff
@@ -26,7 +26,7 @@ Pod::Spec.new do |s|
     s.libraries  = 'c++', 'sqlite3', 'z'
     s.frameworks = 'StoreKit'
 
-    s.dependency 'FirebaseCore', '11.5'
+    s.dependency 'FirebaseCore', '11.6.0'
```

Comment:
This should be "~> 11.6"

Reply:
Should all of these (https://github.com/search?q=repo%3Afirebase%2Ffirebase-ios-sdk+%22%2C+%2711.5.0%27%22&type=code) be `'~> 11.6'`?

Reply:
Will address in a follow-up PR.